src/services/AnalysisArgumentParser.ts

```diff
@@ -180,6 +191,42 @@ class AnalysisArgumentParser {
     });
   }
 
+  addSCMAuditArguments(scriptVersion: string) {
```

Comment:
we should have a ContributorAuditArgumentParser adjacent to the service rather than combining I think - we could create a base argument parser for the shared functionality